features/models.py

```diff
@@ -68,6 +68,12 @@ class Feature(TranslatableModel, TimestampedModel):
         url=models.URLField(
             verbose_name=_("url"), blank=True, help_text=_("URL of the feature")
         ),
+        one_liner=models.CharField(
+            verbose_name=_("one_liner"),
```

Comment:
Nitpicking, but is there a reason to have the underscore in the name? (same question applies to migrations as well)

Reply:
it's like in the line 58 of the same file: source_type or line 84 source_modified_at. I think with an underscore it's more readable than oneliner.

Reply:
`verbose_name` value shouldn't contain a underscore.

Reply:
Not sure if I like _one liner_ as a term. It does have some other meanings as well. Maybe the one mentioned in the ticket (`intro`) would be more suitable.

Reply:
So, "one-liner" would be finer?

Reply:
Yeah, we discussed the term in Slack. I have mixed feelings about both "intro" and "one-liner". I don't think either conveys the purpose in the designs very well.